docs/contributor/arch/001-otel-collector-metric-based-pipeline-status.md

```diff
@@ -0,0 +1,102 @@
+# 1. Trace/Metric Pipeline status based on OTel Collector metrics
+
+Date: 2023-08-11
+
+## Status
+
+Proposed
+
+## Context
+
+The telemetry pipelines deploy active components to a cluster, responsible for dispatching data to third-party services. 
+Inevitably, interruptions can occur, potentially hindering the successful delivery of data. 
+To enhance short-term resilience, we implement retries and buffering mechanisms. 
+Nevertheless, there may be scenarios where data cannot be delivered, requiring user notification for prompt action.
+
+The typical situations causing problems are:
+
+* Connectivity problems with the third-party service.
+* Backpressure or throttling due to overloading the third-party service.
+* Reaching the ingestion limit of a pipeline, which may occur even with auto-scaling in place (subject to a maximum replica setting).
+
+These situations can be monitored by collecting metrics from the relevant components, with [recently added documentation](https://github.com/kyma-project/telemetry-manager/pull/423) to guide the process. 
+However, this approach can be cumbersome for users, as it requires them to identify which services to scrape and filter the relevant metrics. 
+Additionally, these details delve into internal aspects that may undergo changes in the future.
+
+### Goal
+
+* The telemetry manager handles the lifecycle of pipelines and should be the only entity responsible for interpreting the metrics generated by the components.
```

Comment:
I think we call it `telemetry-manager` at other places

Reply:
I checked, it's Telemetry Manager, unified it